### ISSUE TITLE
Upload fails on Captche/Puzzle verification

### ISSUE THREAD

**Issue by phoehnel** [wkaisertexas/tiktok-uploader#169]
This library worked flawlessly for me in headless mode for over a month but since about 1 week i am unable to upload any video.

When disabling headless mode, i can see that the upload halts on a captcha displayed immediately after opening the upload page.
Since the error messages are the same, i assume, this is also what breaks it in headless mode.

Is there already a fix for this?
I already tried to manually solve the captcha on the same device, hoping this would revolve some sort of IP or Account-Block but the Captcha currently turns up on every automated attempt.

______________________

The Captcha looks as follows:
![image](https://github.com/user-attachments/assets/9ac81d81-c12a-4261-aaca-f465fe91e6a6)

And the output is:
```
[13:19:36] Authenticating browser with cookies
[13:19:36] Create a chrome browser instance in headless mode
[13:19:37] Authenticating browser with cookies
[13:19:44] Posting example.mp4
               with description: something
[13:19:44] Navigating to upload page
[13:19:45] Uploading video file
[13:20:46] Removing split window
[13:20:52] Split window not found or operation timed out
[13:20:52] Setting interactivity settings
[13:20:57] Failed to set interactivity settings
[13:20:57] Setting description
[13:21:03] Failed to upload example.mp4
[13:21:03] Message:
Stacktrace:
#0 0x559a035a9c7a <unknown>
#1 0x559a0328ce2c <unknown>
#2 0x559a032d9661 <unknown>
#3 0x559a032d9751 <unknown>
#4 0x559a0331df64 <unknown>
#5 0x559a032fc5ed <unknown>
#6 0x559a0331b303 <unknown>
#7 0x559a032fc363 <unknown>
#8 0x559a032cc247 <unknown>
#9 0x559a032ccb9e <unknown>
#10 0x559a0357024b <unknown>
#11 0x559a035742f1 <unknown>
#12 0x559a0355bafe <unknown>
#13 0x559a03574e52 <unknown>
#14 0x559a0354079f <unknown>
#15 0x559a03599638 <unknown>
#16 0x559a03599810 <unknown>
#17 0x559a035a8dac <unknown>
#18 0x7fc7e8562134 <unknown>
```

**Comment by GregoryWullimann** [comment]
+1

Getting this Captcha at the moment, might be a new security check.

<img width="399" alt="image" src="https://github.com/user-attachments/assets/d20fd7bc-c9d2-4c13-8d0e-48c75d4af06c">

**Comment by G4NS** [comment]
+1
I found this repo(https://github.com/xtekky/TikTok-Captcha-Solver), but I don't know if it works or not

**Comment by GregoryWullimann** [comment]
I tried to use Sadcaptcha (https://www.sadcaptcha.com), it's very easy to implement with this uploader, but unfortunately the solver is bad. Needs multiple tries and I got soft banned after finally positing the video manually (video got 0 views).

An alternative would be https://rapidapi.com/reversecoders/api/flycaptcha but it's crazy expensive if you want to upload just a couple of videos per day.
